examples/03-remote-system/main.py

```diff
@@ -41,7 +41,7 @@ def main():
 
     if args.sync:
         print("Updating the list of countries in Nautobot ...")
-        nautobot.sync_from(local, flags=flags, diff_class=AlphabeticalOrderDiff)
+        nautobot.sync_from(local, flags=flags, diff_class=AlphabeticalOrderDiff, diff=diff)
```

Comment:
`diff` will be an undefined variable if you didn't specify `args.diff`, won't it?